js/pages/tola_management_pages/audit_log/__tests__/auditLog.test.js

```diff
@@ -54,4 +55,25 @@ describe("Audit log", () => {
         }
         expect(renderedLabels).toEqual([ 'Another Disaggregation', 'Type of Leader' ]);
     })
+
+    it("has one component object for indicator only, none for a result element", () => {
+        const component = renderer.create(
+            <IndicatorNameSpan
+                indicator={auditLogIndicator}
+                result_info={null}
+            />
+        ).toJSON();
+        expect(component['children'][2]).toEqual("A test indicator");
+    })
+
+    it("has two components - one for indicator and one for result", () => {
+        const component = renderer.create(
+            <IndicatorNameSpan
+                indicator={auditLogIndicator}
+                result_info={auditLogResult}
+            />
+        ).toJSON();
+        expect(component).toHaveLength(2);
+
+    })
```

Comment:
Is it worth either snapshotting this output or testing for the result date info?  Seems like we could be a little more specific in expecting output here to make sure the date/name string is all correct

Reply:
Done.  I'm never sure when snapshotting is just going to lead to updating the snapshot without looking at it, and when it's useful.